### ISSUE TITLE
Parent - child relation - not mapped correctly

### ISSUE THREAD

**Issue by marek-dziechciarz** [solidify/jira-azuredevops-migrator#1003]
Migrating using config agile (but scrum has also th e same problem).

According to this ticket which is closed: https://github.com/solidify/jira-azuredevops-migrator/issues/985
Statuses are exported improperly.

1. Go to Jira
2. Create Epic (ROM-3644)
3. Create Story in that Epic (ROM-3643)
4. Create Task in that Story (ROM-3645)
5. Create Subtask in that Task (ROM-3646)

After the migration it loos like this:
1. Epic (ROM-3644) Has Parent (!sic) Feature (ROM-3643) - incorrect
2. Feature (ROM-3643) Has parent Story (ROM-3645) - incorrect
3. User Story (ROM-3645) has Childs
3.1. Feature (ROM-3643) - incorrect
3.2. Task (ROM-3646) - correct

Even when I flip Parent and Child config the situation os still not correct.

**Tool version**
 - Version of the jira-azuredevops-migrator tool [3.0.356 ... 

**Attachments**
[jira-export-log-240322-143024.txt](https://github.com/solidify/jira-azuredevops-migrator/files/14723221/jira-export-log-240322-143024.txt)
[config-rome-feature.json](https://github.com/solidify/jira-azuredevops-migrator/files/14723231/config-rome-feature.json)
[wi-import-log-240322-143305.txt](https://github.com/solidify/jira-azuredevops-migrator/files/14723250/wi-import-log-240322-143305.txt)

**Screenshots**
![image](https://github.com/solidify/jira-azuredevops-migrator/assets/56233997/be591b6b-16f5-42db-82ff-71087e0d3c0d)

**Comment by Alexander-Hjelm** [comment]
Not sure right away. Does the Export work? Do the links come along properly? Check the .json files created locally.

**Comment by marek-dziechciarz** [comment]
Nope, there is a error on the export step Story doesn't have link to task as a child because subtask has link to task as child.
So task is exported as a child of subtask not as parent.
Epic and story has proper link

**Comment by Alexander-Hjelm** [comment]
Ok! I will try and reproduce the issue in our test environment. I will let you know as soon as we know more.

**Comment by marek-dziechciarz** [comment]
Cool, especially:
Crete please:
Epic with Story as child (especially created by **Add a child issue** button 
![obraz](https://github.com/solidify/jira-azuredevops-migrator/assets/56233997/dc6eae06-3b03-4e14-a6f7-2ad2e995a3ef)

This Story with Task as a child
![obraz](https://github.com/solidify/jira-azuredevops-migrator/assets/56233997/8428842e-1db4-4c93-bf8c-e7a542f46ce0)

This Task with Sub-Task (ewspecially created by **Create subtask** button
![obraz](https://github.com/solidify/jira-azuredevops-migrator/assets/56233997/5a743dda-e25e-45fe-9f8c-65996b8e7bb0)

**Comment by marek-dziechciarz** [comment]
> Ok! I will try and reproduce the issue in our test environment. I will let you know as soon as we know more.

Any updates on this?

**Comment by Alexander-Hjelm** [comment]
We have been on easter holidays since friday. I have not yet had time to look into this issue, but I am aiming to do it this week.

**Comment by marek-dziechciarz** [comment]
Are there any updates on this?

**Comment by Alexander-Hjelm** [comment]
Hello @marek-dziechciarz! I have not yet had time to look into it yet. Apologies for the delay.

**Comment by marek-dziechciarz** [comment]
Is there a chance to handle this in this week?
Because i must prepare migration, and if there is real chanse to fix this then i will wait, but if not, then i must migrate using relation "releated" instead of parent-child

**Comment by Alexander-Hjelm** [comment]
It is on my todo-list. I have other priorities as well though, so I cannot say how soon I can get around to it.

We do offer professional consultancy hours, through which you can get priority support for bug fixes and feature requests!
